src/roiextractors/imagingextractor.py

```diff
@@ -210,8 +212,13 @@ def get_frames(self, frame_idxs: ArrayType, channel: Optional[int] = 0) -> np.nd
         mapped_frame_idxs = np.array(frame_idxs) + self._start_frame
         return self._parent_imaging.get_frames(frame_idxs=mapped_frame_idxs, channel=channel)
 
-    def get_video(self, start_frame=None, end_frame=None, channel: Optional[int] = 0) -> np.ndarray:
-        assert max(end_frame) < self._num_frames, "'end_frame' range beyond number of available frames!"
```

Comment:
The `end_frame` of `get_video` ought to follow slicing semantics, so it is perfectly safe to request frames beyond the range of the video (it just returns an empty array in that event)

Reply:
Good point.